src/platform/packages/private/kbn-scout-reporting/src/cli/upload_events.ts

```diff
@@ -30,38 +97,13 @@ export const uploadEvents: Command<void> = {
       verifyTLSCerts: SCOUT_REPORTER_ES_VERIFY_CERTS,
     },
     help: `
-    --eventLogPath    (required)  Path to the event log to upload
     --esURL           (required)  Elasticsearch URL [env: SCOUT_REPORTER_ES_URL]
     --esAPIKey        (required)  Elasticsearch API Key [env: SCOUT_REPORTER_ES_API_KEY]
     --verifyTLSCerts  (optional)  Verify TLS certificates [env: SCOUT_REPORTER_ES_VERIFY_CERTS]
+    --eventLogPath    (optional)  Path to an event log file or directory. If omitted, all events in the Scout reports output directory will be uploaded
     `,
   },
   run: async ({ flagsReader, log }) => {
-    // Read & validate CLI options
-    const eventLogPath = flagsReader.requiredString('eventLogPath');
-
-    if (!fs.existsSync(eventLogPath)) {
-      throw createFlagError(`Event log path '${eventLogPath}' does not exist.`);
-    }
-
-    const esURL = flagsReader.requiredString('esURL');
-    const esAPIKey = flagsReader.requiredString('esAPIKey');
-
-    // ES connection
-    log.info(`Connecting to Elasticsearch at ${esURL}`);
-    const es = await getValidatedESClient(
-      {
-        node: esURL,
-        auth: { apiKey: esAPIKey },
-        tls: {
-          rejectUnauthorized: flagsReader.boolean('verifyTLSCerts'),
-        },
-      },
-      { log, cli: true }
-    );
-
-    // Event log upload
-    const reportDataStream = new ScoutReportDataStream(es, log);
-    await reportDataStream.addEventsFromFile(eventLogPath);
+    await runUploadEvents(flagsReader, log);
```

Comment:
This command is now called directly as part of a pipeline script, and we don't want a failure here to affect the final status of the pipeline.

Let's wrap this in a try/catch and just log the error without raising an exception. Just like it was before when the event upload was part of the reporter. 

_Bonus points if you also put this behind an opt-in CLI flag for this behaviour as we'd only want to silence the errors in CI._